src/index.html

```diff
@@ -7,8 +7,46 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;500;700;900&display=swap"
+          rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="nav__wrapper">
+        <div class="nav__logo">
+          <a href="#"><img src="images/logo.png" alt="moyo logo"></a>
+        </div>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" class="nav__link is-active">Apple</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Samsung</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Smartphones</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link"
+                 data-qa="hover">Laptops & Computers
+              </a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Gadgets</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Tablets</a>
+            </li>
```

Comment:
```suggestion
            <li class="nav__item">
              <a href="#" class="nav__link">Gadgets</a>
            </li>
            
            <li class="nav__item">
              <a href="#" class="nav__link">Tablets</a>
            </li>
```

Add an empty line between multiple sibling blocks of html